src/lib/Theme/Theme.docs.mdx

```diff
@@ -19,7 +21,7 @@ You get everything you need out of the box by importing and using these componen
 </script>
 <FontsUrban />
 <Theme>
-  <Button />
+  <button />
```

Comment:
We want this to be uppercase `Button` because its demoing how to use a component, not the standard HTML element.